sbndaq-artdaq-core/Overlays/ICARUS/ICARUSTriggerUDPFragment.hh

```diff
@@ -76,27 +119,132 @@ icarus::ICARUSTriggerInfo icarus::parse_ICARUSTriggerString(const char* buffer)
+      info.wr_name = it->first;
+      info.wr_event_no = std::stol(it->second);
+    }
+    if(it->first.find("sec_WR") != std::string::npos)
+      info.wr_seconds = std::stol(it->second);
+    if(it->first.find("nanosec_WR") != std::string::npos)
+      info.wr_nanoseconds = std::stol(it->second);
+    if(it->first.find("Enable_Type") != std::string::npos)
+      info.enable_type = std::stoi(it->second);
+    if(it->first.find("sec_Enable") != std::string::npos)
+      info.enable_seconds = std::stol(it->second);
+    if(it->first.find("nanosec_Enable") != std::string::npos)
+      info.enable_nanoseconds = std::stol(it->second);
+    if(it->first.find("Gate ID") != std::string::npos && it->first.find("Beam") == std::string::npos && it->first.find("Offbeam") == std::string::npos ) //only store overall gate ID
+      info.gate_id = std::stol(it->second);
+    if(it->first.find("BNB Beam Gate ID") != std::string::npos)
+       info.gate_id_BNB = std::stol(it->second);
+    if(it->first.find("NuMI Beam Gate ID") != std::string::npos)
+      info.gate_id_NuMI = std::stol(it->second);
+    if(it->first.find("Offbeam BNB Beam Gate ID") != std::string::npos)
+      info.gate_id_BNBOff = std::stol(it->second);
+    if(it->first.find("Offbeam NuMI Beam Gate ID") != std::string::npos)
+      info.gate_id_NuMIOff = std::stol(it->second);
+    if(it->first.find("Gate Type") != std::string::npos)
+      info.gate_type = std::stoi(it->second);
+    if(it->first.find("sec_Beam") != std::string::npos)
+      info.beam_seconds = std::stol(it->second);
+    if(it->first.find("nanosec_Beam") != std::string::npos)
+      info.beam_nanoseconds = std::stol(it->second);
+    if(it->first.find("Trigger Type") != std::string::npos)
+      info.trigger_type = std::stoi(it->second);
+    if(it->first.find("Trigger Source") != std::string::npos)
+      info.trigger_source = std::stoi(it->second);
+   
+    //leave these as strings, figure out how to decode them correctly in decoder program. Do they go into binary or a 64-bit unsigned int directly? (I'd think binary for 1 or 0 if given PMT fired)  
+    if(it->first.find("Cryo1 EAST Connector 0") != std::string::npos)
+      info.cryo1_e_conn_0 = it->second;
+    if(it->first.find("Cryo1 EAST Connector 2") != std::string::npos)
+      info.cryo1_e_conn_2 = it->second;
+    if(it->first.find("Cryo2 WEST Connector 0") != std::string::npos)
+      info.cryo2_w_conn_0 = it->second;
+    if(it->first.find("Cryo2 WEST Connector 2") != std::string::npos)
+      info.cryo2_w_conn_2 = it->second;
+
+    //end PMT hex strings
+    if(it->first.find("Cryo1 EAST Counts") != std::string::npos)
+      info.cryo1_east_counts = std::stol(it->second);
+    if(it->first.find("Cryo2 WEST Counts") != std::string::npos)
+      info.cryo2_west_counts = std::stol(it->second);
+      
   }
+  elements.clear();
   */
+  //old implementation preserved in case of emergency, rigid hardcoded version where string must appear in a specific order
```

Comment:
Suggest just removing this block of comments. Now that it's been committed to git, we do have the git entry we could always look back to.